frontend/src/Student/components/Portfolios.js

```diff
@@ -33,7 +33,7 @@ class Portfolios extends Component {
         </NoShowsContainer>
       )
     }
-    return portfolios.map(portfolios => <PortfolioCard key={portfolios.id} portfolio={portfolios} />)
+    return portfolios.map(portfolios => <PortfolioCard key={portfolios.id} portfolio={portfolios} deletePiece={deletePiece}/>)
```

Comment:
There's a decent bit of passing the delete function down to where it gets called.